src/recipe/dto/update-recipe-request.ts

```diff
@@ -14,4 +14,7 @@ export class UpdateRecipeRequest {
 
   @IsOptional()
   preparation: string;
+
+  @IsOptional()
+  isPublic: boolean;
```

Comment:
if property is optional is should also be reflected in the typing:
```suggestion
  isPublic?: boolean;
```